### DISCUSSION TITLE
Missing Features in SimpleSDXL Dev

### DISCUSSION THREAD

**Discussion by DavidDragonsage** [metercai/SimpleSDXL#73] [General]
I hesitate to mention this because likely the coding is in an interim state. But I noticed two things in Comfy Flux mode today that were not in the production code when I was using it yesterday:

1) There is no negative prompt available - so I cannot stop Flux from creating cartoons! 😁

2) The negative styles such as Fooocus Enhance and Fooocus Semi Realistic have lost their weighting factors. This is the actual style:
Name: Fooocus Enhance
Negative Prompt: "(worst quality, low quality, normal quality, lowres, low details, oversaturated, undersaturated, overexposed, underexposed, grayscale, bw, bad photo, bad photography, bad art:1.4), (watermark, signature, text font, username, error, logo, words, letters, digits, autograph, trademark, name:1.2), (blur, blurry, grainy), morbid, ugly, asymmetrical, mutated malformed, mutilated, poorly lit, bad shadow, draft, cropped, out of frame, cut off, censored, jpeg artifacts, out of focus, glitch, duplicate, (airbrushed, cartoon, anime, semi-realistic, cgi, render, blender, digital art, manga, amateur:1.3), (3D ,3D Game, 3D Game Scene, 3D Character:1.1), (bad hands, bad anatomy, bad body, bad face, bad teeth, bad arms, bad legs, deformities:1.3)"

But in the log, this is the result:
Negative
(worst quality, low quality, normal quality, lowres, low details, oversaturated, undersaturated, overexposed, underexposed, grayscale, bw, bad photo, bad photography, bad art), (watermark, signature, text font, username, error, logo, words, letters, digits, autograph, trademark, name), (blur, blurry, grainy), morbid, ugly, asymmetrical, mutated malformed, mutilated, poorly lit, bad shadow, draft, cropped, out of frame, cut off, censored, jpeg artifacts, out of focus, glitch, duplicate, (airbrushed, cartoon, anime, semi-realistic, cgi, render, blender, digital art, manga, amateur), (3D ,3D Game, 3D Game Scene, 3D Character), (bad hands, bad anatomy, bad body, bad face, bad teeth, bad arms, bad legs, deformities)

Without the 1.3 weighting against cartoons in either of these negative styles, I am getting cartoons.

I think I had better go back to the production code and put up with the inconvenience of restarting the program after one or two sets of generations. Its not the end of world! 🙂

Happy programming!

**Comment by metercai** [comment]
The Flux model does not support negative prompt. Even if filled in, it is invalid. So the negative prompt box was canceled under the flux preset. You can solve the problem through prompt or lora.

**Reply by DavidDragonsage** [reply]
I see what you mean. I tried a set of 10 images with and without negatives and there is absolutely no difference. This explains why I have been getting NSFW images from Flux. I will have to figure out how to suppress them in the positive prompt.

Thanks for the information, I will pass this on also. I think a lot of people do not know this. And of course there is no reason to have a negative prompt field if it does not do anything, so that is a good idea to remove it!
